### PR TITLE
RavenDB-20135 handle shard proxy

### PR DESCRIPTION
**Pull request by karmeli87**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-20135

### CODE REVIEW

File: src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs [hunk 1]
```diff
@@ -15,25 +16,26 @@ public ShardedOngoingTasksHandlerProcessorForGetOngoingTasks([NotNull] ShardedDa
...
-                context.Write(writer, result.ToJson());
-            }
+            return new GetOngoingTasksInfoCommand(nodeTag);
         }
 
-        protected override RavenCommand<OngoingTasksResult> CreateCommandForNode(string nodeTag)
+        protected override bool IsCurrentNode(out string nodeTag)
         {
-            return new GetOngoingTasksInfoCommand(nodeTag);
+            nodeTag = GetNodeTag(required: false);
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs:28]: if this is null and you return false from here then what nodeTag we will pass?

Comment (karmeli87) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs:28]: `null` :)

Comment (ppekrol) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs:28]: Let me rephrase myself, what node will be picked then?

Comment (karmeli87) [reply src/Raven.Server/Documents/Sharding/Handlers/Processors/OngoingTasks/ShardedOngoingTasksHandlerProcessorForGetOngoingTasks.cs:28]: the first in the topology of the shard.
e.g. test$1 has the topology of [B,E] -> node `B` will be picked.
